modules/olm-arch-catalog-operator.adoc

```diff
@@ -0,0 +1,25 @@
+// Module included in the following assemblies:
+//
+// * operators/understanding_olm/olm-arch.adoc
+
+[id="olm-arch-catalog-operator_{context}"]
+= Catalog Operator
+
+The Catalog Operator is responsible for resolving and installing CSVs and the
+required resources they specify. It is also responsible for watching
+CatalogSources for updates to packages in channels and upgrading them
+(optionally automatically) to the latest available versions.
+
+A user who wishes to track a package in a channel creates a Subscription
```

Comment:
s/wishes/wants/
Also, you might reword this to address the user in second-person:

> To track a package in a channel, you can create a Subscription resource to configure the desired package, channel, and the CatalogSource you want to pull updates from.

Reply:
Why isn't there a :fire: reaction in GH.